project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -101,4 +101,46 @@ private static User createUser(String email, String nickname) {
 
     return new User(email, nickname, userID);
   }
+
+  private static Document createDocument(String name, String language, String hash, long userID) {
+      Entity docEntity = new Entity("Document");
+      ArrayList<Long> userIDs = new ArrayList<Long>();
+      
+      docEntity.setProperty("name", name);
+      docEntity.setProperty("language", language);
+      docEntity.setProperty("hash", hash);
+      userIDs.add(userID);
```

Comment:
For readability maybe put this line right after you create the array that was you can keep all of the property setting together and make it immediately obvious that you are adding this user to the list

Reply:
Talking about line 112

Reply:
Fixed and changed.